.github/workflows/webapp_check.yml

```diff
@@ -8,10 +8,7 @@ on:
 
 jobs:
   build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [macos-10.15, ubuntu-18.04]
+    runs-on: ubuntu-20.04
```

Comment:
Esto lo haces porque github ha quitado la posibilidad de usar estos macos runners? O es algo en nuestro repo?
Mi PR se rehúsa a arrancar los workflows correspondientes a macos...

Reply:
Yep, a bit of both. We have this #17 open, and the [macOS 10.15](https://github.blog/changelog/2022-07-20-github-actions-the-macos-10-15-actions-runner-image-is-being-deprecated-and-will-be-removed-by-8-30-22/) runner is being deprecated. And this PR adds e2e tests that require Docker Compose, and from what I've seen, the macOS runners don't have Docker installed.